control/config.go

```diff
@@ -232,6 +250,14 @@ func (p *Config) GetPluginConfigDataNodeAll() cdata.ConfigDataNode {
 	return *p.Plugins.All
 }
 
+// IsTLSEnabled returns true if config values enable TLS security
```

Comment:
Maybe it would be better to call it "secure plugin communication" as opposed to "TLS security" - TLS maybe used in the future to secure other communication also